Integrations/ARIA/aria_packet_intelligence.yml

```diff
@@ -0,0 +1,1632 @@
+commonfields:
+  id: ARIA Packet Intelligence
+  version: -1
+name: ARIA Packet Intelligence
+display: ARIA Packet Intelligence
+category: Network Security
+description: 'The ARIA Cybesecurity Solutions Software-Defined Security (SDS)  platform
+  integrates with Demisto to add robustness when responding to incidents. The combination
+  of ARIA hardware, in the form of a Secure Intelligent Adapter (SIA), and software,
+  specifically Packet Intelligence and SDS orchestrator (SDSo), provides the elements
+  required to react instantly when an incident is detected. When integrated with the
+  ARIA solution, you can create playbooks that instruct one or more SIAs to add, modify,
+  or delete rules automatically. These rule changes, which take effect immediately,
+  can block conversations, redirect packets to a recorder or VLAN, or perform a variety
+  of other actions.'
+configuration:
+- display: SDSo Base URL (e.g. http://<IP address or FQDN of SDSo Node>:7443)
+  name: sdso
+  defaultvalue: http://<IP address or FQDN of SDSo Node>:7443
+  type: 0
+  required: true
+- display: Use system proxy
+  name: proxy
+  defaultvalue: 'false'
+  type: 8
+  required: false
+- display: Allow any cert
```

Comment:
```suggestion
- display: Trust any certificate (not secure)
```

Reply:
I am a little confused about the development guide here.https://demisto.developers.paloaltonetworks.com/docs/en/dev-tutorial, If users check this box, means that users will TRUST any certificate. So it is INSECURE, and INSECURE is True. So that in the http request function means that "verify need to set to False", so it is opposite to INSECURE parameter

Reply:
ignore the parameter name if that confuses you, it is like that for historical reasons
eventually, if the box is checked, `verify=False` should be passed to requests